packages/interactivity-router/src/index.ts

```diff
@@ -55,9 +82,10 @@ const fetchPage = async ( url, { html } ) => {
 
 // Return an object with VDOM trees of those HTML regions marked with a
 // `router-region` directive.
-const regionsToVdom = async ( dom, { vdom } = {} ) => {
-	const regions = {};
-	let head;
+const regionsToVdom: RegionsToVdom = async ( dom, { vdom } = {} ) => {
+	const regions = { body: undefined };
+	let head: HTMLElement[];
+	// @ts-ignore
 	if ( process.env.IS_GUTENBERG_PLUGIN ) {
```

Comment:
I had to put a couple of `@ts-ignore` directives so that TS is happy with `process.env`. 

Hopefully once https://github.com/WordPress/gutenberg/pull/61486 is merged we can call `globalThis.IS_GUTENBERG_PLUGIN` and remove the `@ts-ignore`s.